src/screens/DataTable.js

```diff
@@ -275,14 +272,30 @@ export default () => (
               If object is specified 'property' is used to group data by,
               'expand' accepts array of group keys that sets expanded groups and
               'onExpand' is a function that will be called after expand button
-              is clicked with an array of keys of expanded groups.
+              is clicked with an array of keys of expanded groups. If 'onUpdate'
+              is being used to lazy load group items, 'expandable' can be used
+              to specify which items in the data are group header items and
+              'select' and 'onSelect' can be specified in 'groupBy' to help
```

Comment:
```suggestion
              'select' and 'onSelect' can be used to
```

Reply:
I put the groupBy part in there to distinguish it from the select/onSelect props themselves. This is a different one and it's a little confusing in text that they have the same name.